shell/creators/pkg/files/.github/workflows/build-extension.yml

```diff
@@ -64,7 +64,7 @@ jobs:
         shell: bash
         id: build_script
         run: |
-          yarn publish-pkgs -s "${{ github.repository }}" -b main
+          yarn publish-pkgs -s "${{ github.repository }}" -b gh-pages
```

Comment:
Does it have to be the gh-pages branch? Could it be something like `chart` or `extension`?

Reply:
According to the [helm/chart-releaser-action](https://github.com/helm/chart-releaser-action#pre-requisites) it will need to be `gh-pages`. Alternatively the user would have to provide a unique repository url.